engine/src/main/java/org/terasology/world/block/family/SideDefinedBlockFamily.java

```diff
@@ -20,4 +20,5 @@
 
 public interface SideDefinedBlockFamily extends BlockFamily {
```

Comment:
While not a matter for this PR, I don't feel this interface conveys its meaning very well. By side here we actually mean what, the facing direction of the block? Would this conveyed better if it was called SideFacingBlockFamily, and the methods be ```Block getBlockFacing(Side)``` and ```Side getFacingSideOf(Block)```